sherpa/astro/ui/utils.py

```diff
@@ -8551,20 +8539,14 @@ def subtract(self, id=None):
 
         where src_exposure and bg_exposure are the source and
         background exposure times, and src_backscal and bg_backscal
-        are the source and background backscales.  The backscale, read
-        from the ``BACKSCAL`` header keyword of the PHA file [2]_, is
-        the ratio of data extraction area to total detector area.
+        are the source and background backscales.  The backscale, `read
+        from the ``BACKSCAL`` header keyword of the PHA file
+        <https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/node5.html>`_,
+        is the ratio of data extraction area to total detector area.
 
```

Comment:
This isn't rendering nicely

![screenshot](https://user-images.githubusercontent.com/224638/174817271-405e1c14-7352-47d9-bfc9-6ea95664e4a4.png)

I assume it's the fact we have

    blah blah `link text starts but look we have ``SOME KEYWORD`` and more text <actual link>`_ blah bAH

so it's the fact that 

    ``BACKSAL``

is now within a 

    `blah blah`_ 

section.

Reply:
![screenshot](https://user-images.githubusercontent.com/224638/174870298-772d09b9-60ef-4ca2-8711-b8f2d4c4f703.png)

Yay....